sponsor-us.html

```diff
@@ -0,0 +1,236 @@
+        </div>
+        <div
+          class="hidden grid-cols-5 grid-rows-6 grid-flow-col justify-center items-center bg-gray-800 rounded-lg  sm:grid"
+        >
+          <div class="contents text-xl font-extrabold">
+            <div class="list-row"></div>
+            <div class="list-row">Name & Logo on T-Shirts</div>
+            <div class="list-row">Name & Logo on Website</div>
+            <div class="list-row">Name & Logo on Team Banner</div>
+            <div class="list-row">Name & Logo on Robot</div>
+            <div class="break-words rounded-bl-lg list-row">
+              Mention in Interviews & Award Ceremonies
+            </div>
+          </div>
+          <div class="contents text-yellow-800">
+            <div class="flex flex-col list-row">
+              <div class="text-xl font-extrabold">Bronze</div>
+              <div class="text-gray-500">$250 - $499</div>
+            </div>
+            <i class="fas fa-check list-row"></i>
+            <i class="text-red-500 fas fa-times list-row"></i>
+            <i class="text-red-500 fas fa-times list-row"></i>
+            <i class="text-red-500 fas fa-times list-row"></i>
+            <i class="text-red-500 fas fa-times list-row"></i>
+          </div>
+          <div class="contents text-gray-400">
+            <div class="flex flex-col list-row">
+              <div class="text-xl font-extrabold">Silver</div>
+              <div class="text-gray-500">$500 - $1499</div>
+            </div>
+            <i class="fas fa-check list-row"></i>
+            <i class="fas fa-check list-row"></i>
+            <i class="text-red-500 fas fa-times list-row"></i>
+            <i class="text-red-500 fas fa-times list-row"></i>
+            <i class="text-red-500 fas fa-times list-row"></i>
+          </div>
+          <div class="contents text-yellow-400">
+            <div class="flex flex-col list-row">
+              <div class="text-xl font-extrabold">Gold</div>
+              <div class="text-gray-500">$1500 - $2499</div>
+            </div>
+            <i class="fas fa-check list-row"></i>
+            <i class="fas fa-check list-row"></i>
+            <i class="fas fa-check list-row"></i>
+            <i class="fas fa-check list-row"></i>
+            <i class="text-red-500 fas fa-times list-row"></i>
+          </div>
+          <div class="contents text-gray-300">
+            <div class="flex flex-col list-row">
+              <div class="text-xl font-extrabold">Platinum</div>
+              <div class="text-gray-500">$2500+</div>
+            </div>
+            <i class="fas fa-check list-row"></i>
+            <i class="fas fa-check list-row"></i>
+            <i class="fas fa-check list-row"></i>
+            <i class="fas fa-check list-row"></i>
+            <i class="rounded-br-lg fas fa-check list-row"></i>
+          </div>
+        </div>
+      </div>
+    </main>
+
+    <footer class="footer">
+      <div>Made with ♡ by Team 4308 & vidhan</div>
```

Comment:
Can this entire line be removed as it looks unprofessional.

Reply:
Fixed with 639d8ce56fb62db6c4aa404b63b6c8e5a9a17bba.